server/command/command.go

```diff
@@ -32,11 +32,11 @@ const (
 
 // Handler returns API for interacting with plugin commands
 type Handler interface {
-	Handle(args ...string) (*model.CommandResponse, error)
+	Handle(userID string, args ...string) (*model.CommandResponse, error)
```

Comment:
What is the reasoning behind extending the `Handle` func when the userID is available at `command.args.UserId`?